index.html

```diff
@@ -4,10 +4,95 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link href="./style.css" rel="stylesheet">
     <title>Document</title>
 </head>
 
 <body>
+    <header>
+        <div class="navbar">
+            <nav>
+                <img class="logo" src="./img/Takeoff.png">
```

Comment:
Img tag must have an alt attribute. And please, check all your html code with HTML validator, for example, this https://html5.validator.nu/

Reply:
that's fixed